nncf/config_schema.py

```diff
@@ -402,7 +434,7 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
         },
         # TODO: properly search for the initialize in the compression algo list and
         # remove the necessity to piggyback on the RB sparsity algo entry in the config
-        "initializer": INITIALIZER_SCHEMA,
+        "initializer": QUANTIZATION_INITIALIZER_SCHEMA,
```

Comment:
@vshampor is this line still necessary? seems like the initializer schema for RB sparsity was taken from the Quantization one, but there is no initializer schema defined for e.g. Magnitude Sparsity or Filter Pruning. Is the comment above up-to-date?

Reply:
I think it is no longer relevant - we do the initialization of the quantization algo directly during quantization controller creation. The comment may be removed.

Reply:
Thanks, got it.